sns/scripts/proposals/make_sns_action_proposals/make_custom_upgrade_canister_proposal.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+# Set current directory to the directory this script is in
+SCRIPT=$(readlink -f "$0")
+SCRIPT_DIR=$(dirname "$SCRIPT")
+cd $SCRIPT_DIR/../..
+
+# Set env variables based on .env file
+set -o allexport; source .env; set +o allexport
+./utils/setup_env.sh
+
+# Extract the args
+TITLE=$1
+URL=$2
+SUMMARY=$3
+VERSION=$4
+FUNCTION_ID=$5
+CANISTER=$6
+
+cd ./proposals/build_upgrade_canister_proposal
+
+# Locate the WASM
+WASM_PATH='../../../../wasms/'"$CANISTER"'_canister_impl.wasm.gz'
```

Comment:
I think we should pass in the commitId then download it from GitHub

Reply:
I think its less error prone that way.

Reply:
Ahh we don't currently have a script for downloading a single wasm. I'll make one tomorrow.